server/data/data-library/src/main/java/org/apache/james/domainlist/lib/AbstractDomainList.java

```diff
@@ -131,7 +131,7 @@ private boolean mayChangeDefaultDomain() {
         return Domain.LOCALHOST.equals(defaultDomain);
     }
 
-    private void setDefaultDomain(Domain defaultDomain) throws DomainListException {
+    protected void setDefaultDomain(Domain defaultDomain) throws DomainListException {
```

Comment:
I would rather provide a `configure` method that takes a POJO, publishing setters looks like a bad idea